cosine_matcher.py

```diff
@@ -27,54 +25,67 @@ def __init__(self, encoding='utf-8', analyzer='word', ngram_range=(1,1), \
 
         This code assumes that you have already cleaned & processed the text corpus. 
 
-        '''
+        SAMPLE USAGE:
+        searcher = CosineMatcher()
+        searcher.train('C:/Users/User/Desktop/myFile.csv')
+        searcher.find('manila irrigation')
+        >>> {'manila north rd ...
+        """
         self.match_corpus = None
         self.matrix = None 
         self.vectorizer = TfidfVectorizer(encoding=encoding, analyzer=analyzer,\
             ngram_range=ngram_range, min_df=min_df, max_df=max_df,\
             use_idf=use_idf)
 
 
-    def train(self, corpus, train_on='training_col'):
-        '''
+    def train(self, corpus, train_on='searchspace'):
```

Comment:
rename to `create_matrix` ? `train` is not a good name for a function that's NOT training a ML model 

rename `train_on` as well